dhcpd.conf

```diff
@@ -0,0 +1,106 @@
+#
```

Comment:
Where did this file originate?

Reply:
It comes when we install isc-dhcp-server. this file is placed into /etc/dhcp directory. Its not present before the installation

Reply:
Instead of copying this file, we can put our configuration in our own file and include that in the system file with
` include "filename";`

This is analogous to how we reworked the interfaces file.

see https://linux.die.net/man/5/dhcpd.conf

Reply:
The reason i did not do that was because there will still be loads of sed commands to comment out and uncomment stuff in the original file, to add the source file info in the file and to delete the info if already present. Its not going to make out config file simpler. it will just add the feature of importing configurations.

Reply:
I think you are misunderstanding or confused about the situation. The setup required is just as trivial as what we have now:

replace
`cp conffile /etc/dhcp/dhcpd.conf`
use 
`echo 'include "conffile";' >> /etc/dhcp/dhcpd.conf`

Reply:
the dhcpd.conf file we get after we download isc-dhcp-server is different. it has a lot of example code and wrong configuration just to demonstrate stuff. that is why inittially the server startup fails because that file does not have any network specific definitions. it just has a lot of example lines that dont work. I remove those lines and add our configuration. so the file you see now is the polished version. for us to include the configuration, its not just a one line command. there will be more where i need to delete or comment the useless lines before adding our configuration in so that there are no duplicates

Reply:
we no longer need this file correct?